tests/testthat/test-endpoints-download.R

```diff
@@ -0,0 +1,91 @@
+context("endpoints-download")
+
+test_that("indicator download returns bytes", {
+  test_redis_available()
+  test_mock_model_available()
+
+  ## Setup payload
+  path <- setup_submit_payload()
+
+  ## Run the model
+  queue <- hintr:::Queue$new()
+  model_submit <- submit_model(queue)
+  response <- model_submit(readLines(path))
+  expect_true("id" %in% names(response))
+
+  testthat::try_again(4, {
+    Sys.sleep(2)
+    summary <- download_summary(queue)
+    download <- summary(response$id)
+    expect_type(download$bytes, "raw")
+    expect_length(download$bytes, file.size(
+      system.file("output", "malawi_summary_download.zip", package = "hintr")))
+    expect_equal(download$id, response$id)
+  })
+})
+
+test_that("spectrum download returns bytes", {
+  test_redis_available()
+  test_mock_model_available()
+
+  ## Setup payload
+  path <- setup_submit_payload()
+
+  ## Run the model
+  queue <- hintr:::Queue$new()
+  model_submit <- submit_model(queue)
+  response <- model_submit(readLines(path))
+  expect_true("id" %in% names(response))
+
+  testthat::try_again(4, {
+    Sys.sleep(2)
```

Comment:
What is the wait for here? If it's the model running we might want to use the queue's `wait` method for an optimal wait (and elsewhere with `Sys.sleep`)